tests/unit/entityInfo.spec.ts

```diff
@@ -6,43 +6,76 @@ import Vuetify from 'vuetify'
 import { getVuexStore } from '@/store'
 
 // Components
-import { mount } from '@vue/test-utils'
+import { createLocalVue, mount } from '@vue/test-utils'
 import { EntityInfo } from '@/components/common'
 
-Vue.use(Vuetify)
+// Other
+import mockRouter from './MockRouter'
+import VueRouter from 'vue-router'
 
+Vue.use(Vuetify)
 const vuetify = new Vuetify({})
-
-describe.skip('Entity Info component with an NR', () => {
-  let wrapper: any
-  let store: any
+document.body.setAttribute('data-app', 'true')
+
+describe('Entity Info component in a Correction as a named company', () => {
+  let wrapper
+  let store
+
+  const mockFiling = {
+    header: {
+      name: 'incorporationApplication',
+      filingId: 12345
+    },
+    business: {
+      identifier: 'BC1234567',
+      legalType: 'BEN'
+    },
+    incorporationApplication: {
+      contactPoint: {
+        email: 'mock@email.com',
+        phone: '123-456-7890'
+      },
+      nameRequest: {
+        legalType: 'BEN',
+        nrNumber: 'NR 1234567',
+        legalName: 'My Mock Name Inc.'
+      },
+      offices: {},
+      parties: [],
+      shareClasses: [],
+      incorporationAgreement: {}
+    }
+  }
 
   beforeEach(() => {
     store = getVuexStore()
```

Comment:
`store` only needs to be called once, which you can do above, near where you get `vuetify`.

Reply:
fixed in next commit!